lib/keypair.ex

```diff
@@ -35,8 +39,9 @@ defmodule Elixium.KeyPair do
   defp create_keyfile({public, private}) do
     if !File.dir?(".keys"), do: File.mkdir(".keys")
 
-    pub_hex = Base.encode16(public)
-    File.write(".keys/#{pub_hex}.key", private)
+    address = address_from_pubkey(public)
+
+    File.write(".keys/#{address}.key", private)
```

Comment:
It may be worth switching to ```File.write!``` so we can catch issues of the key not writing, however, this will have to be refactored with the os selection at a later date anyway.